Lib/test/test_dataclasses.py

```diff
@@ -2926,24 +2926,49 @@ class C:
                 x: int
 
     def test_generated_slots_value(self):
+
+        class Root:
+            __slots__ = {'x'}
+
+        class Root2(Root):
+            __slots__ = 'aa'
+
         @dataclass(slots=True)
-        class Base:
-            x: int
+        class Base(Root2):
+            y: int
 
-        self.assertEqual(Base.__slots__, ('x',))
+        self.assertEqual(Base.__slots__, ('y', ))
 
         @dataclass(slots=True)
         class Delivered(Base):
```

Comment:
It's probably not worth changing here, but surely this class was orginally meant to be named `Derived`. And below `AnotherDerived`.

Reply:
I can't resist fixing this either, done.